pages/dkp/kaptain/2.0.0/install/deploy-kaptain/index.md

```diff
@@ -0,0 +1,103 @@
+excerpt: Deploy Kaptain on air-gapped and non air-gapped environments
+beta: false
+enterprise: false
+---
+
+<p class="message--warning"><strong>WARNING: </strong>
+Kaptain deploys to all clusters in your selected Workspace. If you do not want to deploy Kaptain to a certain cluster, you must move it to another Workspace.
+</p>
+
+## Enable and deploy Kaptain using the DKP UI
+
+Follow these steps to enable Kaptain in air-gapped and non air-gapped environments from the DKP UI:
+
+1.  **Enterprise only**: From the top menu bar, select your target Workspace.
+
+1.  From the sidebar menu, select **Applications**.
+
+1.  Type Kaptain in the **Applications** search bar. If Kaptain is not available in the UI, [add Kaptain to your DKP catalog][add_Kaptain] via CLI first.
+
+1.  From the Kaptain tile, select the three dot menu > **Enable**.
+    The `Enable Workspace Catalog Application` page is displayed.
+
+1.  Verify that you deploy to the correct target Workspace and Clusters.
+    - If the Workspace is incorrect, go back to the main dashboard and select the correct Workspace as in step 1.
+    - If you don’t want to deploy Kaptain to all clusters, interrupt deployment and manually move the clusters where you don’t want to deploy Kaptain to another Workspace.
+
+1.  If available, select a version from the drop-down menu. This drop-down menu will only be visible if there is more than one version.
+
+1.  (Optional) If you want to override the default configuration values, copy your customized values into the text editor under **Configure Service** or upload your yaml file that contains the values:
+
+    ```yaml
+    someField: someValue
+    ```
+
+1.  Confirm the details are correct, and then select the **Enable** button to enable and trigger deployment.
+    The status changes to **Enabled**.
+
+1.  If you want to deploy Kaptain to additional Workspaces, repeat these steps for each additional Workspace.
+
+Alternately, you can use the [CLI](#enable-and-deploy-Kaptain-using-the-dkp-cli) to enable your catalog applications.
+
+### Verify the status of deployment using the DKP UI
+
+Follow these steps to verify the deployment of Kaptain:
+
+1.  From the sidebar menu, select **Clusters**.
+
+1.  From your clusters tile, select **View Details**.
+
+1.  Select the **Applications** tab and scroll down to find the Kaptain tile.
+
+1.  If Kaptain was deployed successfully, the status is **Deployed**.
+
+<p class="message--note"><strong>NOTE: </strong>Depending on the number of clusters, it can take several minutes until provisioning is finished and status is changed to <b>Deployed</b>.</p>
+
+## Enable and deploy Kaptain using the DKP CLI
+
+Follow these steps to enable Kaptain in air-gapped and non air-gapped environments from the DKP CLI:
+
+1.  Enable Kaptain to deploy to [your existing Managed and Attached clusters][existcluster] with an `AppDeployment` resource.
+
+<!-- why are we referring to attached clusters only and not to managed? -->
```

Comment:
I borrowed and adapted this section from https://docs.d2iq.com/dkp/kommander/2.2/workspaces/applications/catalog-applications/application-deployment/#enable-the-application-using-the-cli 
but I don't understand why we are only talking about attached clusters there. 
Since workspaces can have both attached and managed clusters, can we switch keep the sentence above?

Reply:
I think technically we should refer to both. The "happy path" scenario is that Kommander is installed in a management cluster and that is used to deploy apps to managed clusters. This was probably written with that in mind. But we also support deploying to the management cluster.